bait-files/src/main/webapp/results.js

```diff
@@ -0,0 +1,125 @@
+const STAR_FILLED = '★';
+const STAR_OUTLINE = '☆';
+const MAX_RATING = 5;
+
+const STAR_RATING = 'star-rating';
+const AUTHENTICITY_EXPLANATION = 'authenticity-explanation';
+
+const DOES_NOT_EXIST = 'It is likely that this person does not exist.';
+const EXISTS = 'It is likely that this person exists.';
+
+const PHONE_INPUT = 'phone-input-container';
+const NAME_INPUT = 'name-input-container';
+const USERNAME_INPUT = 'username-input-container';
+const EMAIL_INPUT = 'email-input-container';
+const IMAGE_INPUT = 'image-input-container';
+const ADDRESS_INPUT = 'address-input-container';
+const NOTES_INPUT = 'notes-input-container';
+
+const PHONE_RESULTS = 'phone-results-container';
+const NAME_RESULTS = 'name-results-container';
+const USERNAME_RESULTS = 'username-results-container';
+const EMAIL_RESULTS = 'email-results-container';
+const IMAGE_RESULTS = 'image-results-container';
+const ADDRESS_RESULTS = 'address-results-container';
+const REVIEWER_NOTES = 'reviewer-notes-container';
+
+const VALID = 'VALID';
+const INVALID = 'INVALID';
+const NOT_PROVIDED = 'This information was not provided.';
+
+const INVALID_STRING =
+    'There were some problems validating the provided information.';
+const VALID_STRING =
+    'There were no problems validating the provided information.';
+
+function getReview() {
+  const queryString = window.location.search;
+  const params = new URL(location.href).searchParams;
+  const requestId = params.get('reviewId');
+  if (requestId == null || isNaN(requestId)) {
+    window.location.replace('/');
+    return;
+  }
+
+  fetch('/review' + queryString)
+      .then(response => response.json())
+      .then((result) => {
+        document.getElementById(NAME_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.name)));
+        document.getElementById(EMAIL_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.email)));
+        document.getElementById(USERNAME_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.username)));
+        document.getElementById(PHONE_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.phoneNum)));
+        document.getElementById(ADDRESS_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.address)));
+        document.getElementById(IMAGE_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.image)));
+
+        let stars = '';
+        for (let i = 0; i < MAX_RATING; i++) {
+          if (i < result.review.authenticityRating) {
+            stars += STAR_FILLED;
+            continue;
+          }
+          stars += STAR_OUTLINE;
+        }
```

Comment:
I *think* you might just be able to do

```suggestion
        let starRatingString = 
            STAR_FILLED.repeat(result.review.authenticityRating)
            .concat(STAR_OUTLINE.repeat(MAX_RATING - result.review.authenticityRating))
```